src/main/java/at/hannibal2/skyhanni/utils/TimeUtils.kt

```diff
@@ -142,6 +148,13 @@ object TimeUtils {
     }
 
     fun getCurrentLocalDate(): LocalDate = LocalDate.now(ZoneId.of("UTC"))
+
+    fun getSeasonByName(input: String): String {
```

Comment:
Makethe funtion return a season element or null and move this function to the same spot you have the enum.

Reply:
done